var.tfvars

```diff
@@ -37,7 +37,7 @@ installer_log_level         = "info"
 ansible_extra_options       = "-v"
 
 #helpernode_tag             = "fddbbc651153ef2966e5cb4d4167990b31c01ceb"
-#install_playbook_tag       = "fd018e391831d431a0828b58c7b25f7c5b0bb581"
+#install_playbook_tag       = "02a598faa332aa2c3d53e8edd0e840440ff74bd5"
```

Comment:
@Prajyot-Parab can you just make the tags an empty string here ? We are anyway making it default in variables.tf. 
Since this is a minor patch I'm asking you to make this change in this PR itself